sql/npc_list.sql

```diff
@@ -22574,22 +22455,22 @@ INSERT INTO `npc_list` VALUES (17613278,'HomePoint#2','Home Point #2',0,102.000,
-INSERT INTO `npc_list` VALUES (17617182,'Treasure_Casket','Treasure Casket',0,0.000,0.000,0.000,7,40,40,0,4,4,2,3,0x0000C50300000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17617183,'Treasure_Casket','Treasure Casket',0,0.000,0.000,0.000,7,40,40,0,4,4,2,3,0x0000C50300000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17617184,'Treasure_Casket','Treasure Casket',0,0.000,0.000,0.000,7,40,40,0,4,4,2,3,0x0000C50300000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17617185,'Treasure_Casket','Treasure Casket',0,0.000,0.000,0.000,7,40,40,0,4,4,2,3,0x0000C50300000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17617186,'Treasure_Casket','Treasure Casket',0,0.000,0.000,0.000,7,40,40,0,4,4,2,3,0x0000C50300000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17617187,'Treasure_Casket','Treasure Casket',0,0.000,0.000,0.000,7,40,40,0,4,4,2,3,0x0000C50300000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17617188,'Treasure_Casket','Treasure Casket',0,0.000,0.000,0.000,7,40,40,0,4,4,2,3,0x0000C50300000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17617189,'Treasure_Casket','Treasure Casket',0,0.000,0.000,0.000,7,40,40,0,4,4,2,3,0x0000C50300000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17617190,'Treasure_Casket','Treasure Casket',0,0.000,0.000,0.000,7,40,40,0,4,4,2,3,0x0000C50300000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17617191,'Treasure_Casket','Treasure Casket',0,0.000,0.000,0.000,7,40,40,0,4,4,2,3,0x0000C50300000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17617192,'Treasure_Casket','Treasure Casket',0,0.000,0.000,0.000,7,40,40,0,4,4,2,3,0x0000C50300000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17617193,'Treasure_Casket','Treasure Casket',0,0.000,0.000,0.000,7,40,40,0,4,4,2,3,0x0000C50300000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17617194,'Treasure_Casket','Treasure Casket',0,0.000,0.000,0.000,7,40,40,0,4,4,2,3,0x0000C50300000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17617195,'Treasure_Casket','Treasure Casket',0,0.000,0.000,0.000,7,40,40,0,4,4,2,3,0x0000C50300000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17617196,'Treasure_Casket','Treasure Casket',0,0.000,0.000,0.000,7,40,40,0,4,4,2,3,0x0000C50300000000000000000000000000000000,0,NULL,0);
-INSERT INTO `npc_list` VALUES (17617197,'Treasure_Casket','Treasure Casket',0,0.000,0.000,0.000,7,40,40,0,4,4,2,3,0x0000C50300000000000000000000000000000000,0,NULL,0);
+-- NC: INSERT INTO `npc_list` VALUES (17617182,'NOT_CAPTURED','Treasure Casket',0,0.000,0.000,0.000,0,50,50,0,0,0,0,0,0x0000320000000000000000000000000000000000,0,NULL,0);
```

Comment:
I think you need to rebase onto current master when you get a chance, and make sure unrelated NPC's aren't accidentally being reverted.

Reply:
doh! Yeah, I can fix that.

Reply:
This should be fixed now.